### PR TITLE
Hub and spoke as tiles example

### PR DESCRIPTION
**Pull request by halocline**
NOTE: This PR is to merge into the nav-patterns branch

[Preview](https://deploy-preview-751--keen-mayer-a86c8b.netlify.app/templates/hub-and-spoke-navigation#hub-&-spoke-as-tiles)

Hub and Spoke as tiles example should demonstrate:

- [x] Pattern | Hub presenting spokes which lead to either:
  - [x] A subsequent Hub presenting more spokes
  - [x] Detail page
- [x] Each step down provides navigational context back up the information hierarchy
- [x] Each page has a primary focus with minimal contextual navigation chrome
- [x]  Desktop vs. Mobile
- [x]  Copy-pasteable

### CODE REVIEW

File: aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+  return (
+    <Header
+      fill="horizontal"
+      height="xsmall"
+      pad={{ horizontal: size !== 'small' ? 'large' : 'medium' }}
+    >
+      <AppIdentity
+        name={`Service ${
+          typeof currentPage.parent !== 'undefined'
+            ? `| ${currentPage.parent}`
```
Comment (taysea) [inline aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js:223]: This breadcrumb type layout was a little unexpected to me -- I hadn't noticed the title changed so it took me a while to figure out how to go back up one-level. I guess I was expecting more of a "back to [parent page name]" button just above the page content.

Screen Shot 2020-05-06 at 9.25.40 AM

File: aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+    parent: 'Server Cluster',
+  },
+  {
+    name: 'Beacon A',
+    icon: <LocationPin />,
+    layout: 'panes',
+    parent: 'Beacon Cluster',
+  },
+  {
+    name: 'Beacon B',
```
Comment (taysea) [inline aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js:121]: I noticed that when I click on Devices, the Beacon B card gets automatically focused for some reason:
Screen Shot 2020-05-06 at 9 25 36 AM

Comment (taysea) [reply aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js:121]: Seems like the page is reading that card as the same card I had just clicked on or something?

File: aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js (hunk 1 above)
Comment (halocline) [reply aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js:223]: Agree. I was trying to apply this pattern that was submitted in the #hpexs-designreviewers channel, but I too feel like it gets lost. https://hpe.slack.com/archives/CTUQ1PQDS/p1585679610002700

https://designer.grommet.io/dashboard?id=GLC-Page-Naming-and-Breadcrumbs-edgar-circenis-hpe-com

I like what we are doing on the site better:
Screen Shot 2020-05-06 at 10 35 40 AM

Comment (halocline) [reply aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeTilesExample.js:223]: Will adjust.

File: aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+  }, [currentPage]);
+
+  const { children, layout, name, parent } = currentPage;
+
+  return (
+    <ResponsiveContext.Provider value={mobile && 'small'}>
+      <AppContainer>
+        <AppHeader currentPage={currentPage} setCurrentPage={setCurrentPage} />
+        {typeof parent !== 'undefined' && (
+          <SimpleButton onClick={() => setCurrentPage(getPageDetails(parent))}>
```
Comment (taysea) [inline aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js:162]: Should we wrap this in a Box with align="start" so the button width doesn't span the whole screen?
Notice here on hover:
Screen Shot 2020-05-06 at 4 27 23 PM
Good news is that it appeared on Chrome, Safari, and Firefox

Comment (halocline) [reply aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js:162]: Yes, we should. Could you do me a favor and try that in your local environment? When I did that earlier, I was having similar funny business to what I was showing you earlier -- particularly in pages which the tiles / lists overflow.

File: aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+      currentPage.children.map(child =>
+        pages.find(page => page.name === child),
+      );
+    setSubPages(pageDetails);
+  }, [currentPage]);
+
+  const { children, layout, name, parent } = currentPage;
+
+  return (
+    <ResponsiveContext.Provider value={mobile && 'small'}>
```
Comment (britt6612) [inline aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js:158]: one thing to note is that once my example PR is merged I moved this `  <ResponsiveContext.Provider value={mobile && 'small'}>` to be within `example.js` file so we wont need this here

File: aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js (hunk 3 above)
Comment (britt6612) [reply aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js:162]: not sure which issue you guys are referring to but in my local I did `alignSelf="start"` and didnt see any issues

File: aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js (hunk 4 above)
Comment (britt6612) [reply aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js:158]: Also not seeing a big change for mobile maybe Im missing something ?

Comment (halocline) [reply aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js:158]: I messed that up when merging master into this branch. Sorry!

Comment (halocline) [reply aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js:158]: > one thing to note is that once my example PR is merged I moved this ` <ResponsiveContext.Provider value={mobile && 'small'}>` to be within `example.js` file so we wont need this here

Yay!

File: aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js (hunk 3 above)
Comment (halocline) [reply aries-site/src/examples/templates/hub-spoke-navigation/HubSpokeCardsExample.js:162]: Nice! I will try that in the AM.

**Comment by halocline** [comment]
> Looks good! just fix merge conflicts

in progress now ;)

**Pull request merged**